code/script.js

```diff
@@ -251,93 +265,202 @@ const setSecret = () => {
 
 // This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
+  
+// Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  
+// What else should happen when we start the game?
+generateBoard()
+setSecret()
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
-
+// This variable stores what option group (category) the question belongs to.
+  
+// We also need a variable that stores the actual value of the question we've selected.
+const value = questions.value
+console.log(value)
```

Comment:
remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️